Forge.TreeWalker.UnitTests/test/TreeWalkerUnitTests.cs

```diff
@@ -33,27 +33,58 @@ public class TreeWalkerUnitTests
 
         private Guid sessionId;
         private IForgeDictionary forgeState = new ForgeDictionary(new Dictionary<string, object>(), Guid.Empty, Guid.Empty);
-        private ITreeWalkerCallbacks callbacks;
+        private ITreeWalkerCallbacksV2 callbacksV2;             // 20201012, switch to V2 callback.
```

Comment:
>             // 20201012, switch to V2 callback. [](start = 51, length = 48)

Please remove this comment. I don't think timestamped comments when code was changed are helpful.

Reply:
removed.

---
In reply to: [505915856](https://github.com/microsoft/Forge/pull/32#discussion_r505915856) [](ancestors = 505915856)